.github/ISSUE_TEMPLATE/1-add-site-with-2fa.md

```diff
@@ -0,0 +1,62 @@
+---
+name: 🔒 Add site with 2FA
+about: Request that a site that supports 2FA be added.
+title: Add [site name]
+labels: add site
+assignees: ''
+
+---
+
+<!-- Before submitting this issue, please update the title to include the name of the site to be added.
+Submit a single issue for each site to be added.
+
+In Markdown, checkboxes work like this:
+- [ ] Unchecked box.
+- [x] Checked box.
+
+Check both boxes below before submitting your issue to verify that you have already checked for duplicate issues and pull requests relating to your request. -->
+
+- [ ] I have searched both open and closed [issues](https://github.com/2factorauth/twofactorauth/issues). The issue I'm creating is not a duplicate of an existing issue.
+- [ ] I have searched both open and closed [pull requests](https://github.com/2factorauth/twofactorauth/pulls). The issue I'm creating is not a duplicate of an existing pull request.
+
+### Information about the site to be added that supports 2FA: ###
+<!-- Official name of the site -->
+* Site name - ``
+
+<!-- Link to the main page -->
+* Site URL - ``
+
+* Does the site support 2FA? - `Yes`
+
+<!-- Link to documentation on how to enable 2FA on the site.
+Attach screenshots of the setup/login process if no public-facing documentation link is available, redacting any personal information. -->
+* Documentation URL - ``
+
+### Information about what methods of 2FA the site supports: ###
+<!-- Mark each implementation of 2FA that the site supports.
+See our [wiki](https://github.com/2factorauth/twofactorauth/wiki/FAQ-2FA-Types) for more information about different types of 2FA implementations. -->
+
+* Software:
+  - [ ] TOTP (RFC-6238, Google Authenticator)
+  - [ ] Proprietary (Authy, Duo, etc.)
+
+* Hardware:
+  - [ ] U2F keys
+  - [ ] WebAuthn
```

Comment:
```suggestion
```
Does it make sense to distinguish "WebAuthn" from others if we don't have a dedicated `tfa` value for it?
If it's just for future enhancements, we could also include `biometric` then (and map it to `hardware`).

Reply:
I'm guessing the original idea behind including it as a method in the template was to allow people to mark either U2F or WebAuthn and have us map both to `U2F` at the moment. We could also change it to "U2F/WebAuthn" or "FIDO2/U2F".

Reply:
I'm afraid the users that use an issue template (as opposed to a pull request) are more likely _not_ the most tech-savvy ones, i.e. they might struggle to distinguish these in the first place.
Also, WebAuthn doesn't enforce a hardware solution, although most uses will probably include a hardware dongle of some sort.
In short: I'd go for a combined entry `FIDO2 / U2F keys`.

Reply:
Doesn’t FIDO2 include non-2FA authentication?

Reply:
I agree on that front. I think matching the option to what a service typically would state in their documentation is the best way to go.

For example, [Dropbox](https://help.dropbox.com/teams-admins/team-member/enable-two-step-verification) states it as 

>A security key is a small USB, Bluetooth, or Near Field Communication (NFC) device that follows one of the open standards:
>‘FIDO Universal 2nd Factor (U2F)'
>'Web Authentication (WebAuthn)', also known as 'FIDO2’

and [Google](https://support.google.com/accounts/answer/6103523) calls it

> a security key that plugs into a USB port and works with FIDO Universal 2nd Factor (U2F) or FIDO2

so by listing the option as "FIDO2 / U2F security keys", I think we'd capture most of the terms used in end-user facing documentation.